opensearch/descriptions/datasets.xml

```diff
@@ -14,13 +14,14 @@
   projects the datasets are related to), and the result set.</Description>
   <Tags></Tags>
   <Contact>Han Wang</Contact>
-  <Url type="application/json" rel="dco:YearsQuery" template="http://data.deepcarbon.net/dco-s2s/opensearch/services/datasets/search.php?request=years&amp;communities={dco:Communities?}&amp;groups={dco:Groups?}&amp;authors={dco:Authors?}&amp;other_authors={dco:OtherAuthors?}&amp;projects={dco:Projects?}" />
-  <Url type="application/json" rel="dco:CommunitiesQuery" template="http://data.deepcarbon.net/dco-s2s/opensearch/services/datasets/search.php?request=communities&amp;years={dco:Years?}&amp;groups={dco:Groups?}&amp;authors={dco:Authors?}&amp;other_authors={dco:OtherAuthors?}&amp;projects={dco:Projects?}" />
-  <Url type="application/json" rel="dco:GroupsQuery" template="http://data.deepcarbon.net/dco-s2s/opensearch/services/datasets/search.php?request=groups&amp;communities={dco:Communities?}&amp;years={dco:Years?}&amp;authors={dco:Authors?}&amp;other_authors={dco:OtherAuthors?}&amp;projects={dco:Projects?}" />
-  <Url type="application/json" rel="dco:AuthorsQuery" template="http://data.deepcarbon.net/dco-s2s/opensearch/services/datasets/search.php?request=authors&amp;communities={dco:Communities?}&amp;groups={dco:Groups?}&amp;years={dco:Years?}&amp;other_authors={dco:OtherAuthors?}&amp;projects={dco:Projects?}" />
-  <Url type="application/json" rel="dco:OtherAuthorsQuery" template="http://data.deepcarbon.net/dco-s2s/opensearch/services/datasets/search.php?request=other_authors&amp;communities={dco:Communities?}&amp;groups={dco:Groups?}&amp;authors={dco:Authors?}&amp;years={dco:Years?}&amp;projects={dco:Projects?}" />
-  <Url type="application/json" rel="dco:ProjectsQuery" template="http://data.deepcarbon.net/dco-s2s/opensearch/services/datasets/search.php?request=projects&amp;communities={dco:Communities?}&amp;groups={dco:Groups?}&amp;authors={dco:Authors?}&amp;other_authors={dco:OtherAuthors?}&amp;years={dco:Years?}" />
-  <Url type="text/html" rel="dco:ResultsListQuery" template="http://data.deepcarbon.net/dco-s2s/opensearch/services/datasets/search.php?request=datasets&amp;communities={dco:Communities?}&amp;groups={dco:Groups?}&amp;authors={dco:Authors?}&amp;other_authors={dco:OtherAuthors?}&amp;projects={dco:Projects?}&amp;years={dco:Years?}&amp;limit={count}&amp;offset={startIndex}" />
+  <Url type="application/json" rel="dco:YearsQuery" template="http://data.deepcarbon.net/dco-s2s/opensearch/services/datasets/search.php?request=years&amp;communities={dco:Communities?}&amp;groups={dco:Groups?}&amp;authors={dco:Authors?}&amp;other_authors={dco:OtherAuthors?}&amp;projects={dco:Projects?}&amp;datatypes={dco:DataType?}" />
```

Comment:
In that case you will also need to update datasets.html:43 to use 'Datatype' instead of 'Datatypes'.